### PR TITLE
[Synapse] Spark Configuration cmdlets

### PR DESCRIPTION
**Pull request by yanjungao718**
## Description

Add Synapse Spark Configuration related cmdlets:
`New-AzSynapseSparkConfiguration`
`Get-AzSynapseSparkConfiguration`
`Export-AzSynapseSparkConfiguration`
`Remove-AzSynapseSparkConfiguration`

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1057
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by BethanyZhou** [comment]
@yanjungao718 , please fix  signature issue first:  
```
"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
  "Az.Synapse","Microsoft.Azure.Commands.Synapse.ExportAzureSynapseSparkConfiguration","Export-AzSynapseSparkConfiguration","1","8100","Export-AzSynapseSparkConfiguration Does not support ShouldProcess but the cmdlet verb Export indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
```

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: src/Synapse/Synapse/Az.Synapse.psd1 [hunk 1]
```diff
@@ -298,7 +304,12 @@ PrivateData = @{
     - Added ''Get-AzSynapseSqlScript'' cmdlet
     - Added ''Remove-AzSynapseSqlScript'' cmdlet
     - Added ''Export-AzSynapseSqlScript'' cmdlet
-    - Added ''Set-AzSynapseSqlScript'' cmdlet'
+    - Added ''Set-AzSynapseSqlScript'' cmdlet
```
[the remark this answers is not in the document]
Comment (yanjungao718) [reply src/Synapse/Synapse/Az.Synapse.psd1]: updated, please help to review, thanks.

File: src/Synapse/Synapse/ChangeLog.md [hunk 2]
```diff
@@ -21,6 +21,11 @@
 ## Upcoming Release
 * Renamed parameter FolderName in `Set-AzSynapseSqlScript` to FolderPath and keeped FolderName as alias
 * Updated `Set-AzSynapseNoteBook` and `Set-AzSynapseSparkJobDefinition` to support new parameter [-FolderPath]
+* Added cmdlets for Synapse Spark Configuration
+    - Added `Get-AzSynapseSparkConfiguration` cmdlet
+	- Added `New-AzSynapseSparkConfiguration` cmdlet
+    - Added `Export-AzSynapseSparkConfiguration` cmdlet
+	- Added `Remove-AzSynapseSparkConfiguration` cmdlet
```
[the remark this answers is not in the document]
Comment (yanjungao718) [reply src/Synapse/Synapse/ChangeLog.md]: updated, please help to review, thanks.

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**